t/python/t0005-kvs.py

```diff
@@ -39,7 +40,9 @@ def set_and_check_context(self, key, value, type):
         kd2 = flux.kvs.KVSDir(self.f)
         nv = kd2[key]
         if isinstance(value, bytes) and type is str:
-            self.assertEqual(value.decode('utf-8'), nv)
+            self.assertEqual(value.decode("utf-8"), nv)
```

Comment:
I think normally these files are not subject to formatting by `black`. That might be why the formatting changed Did you run `black` directly on the file?

Actually, it could be a mistake that the python test files aren't being formatted. The `.pre-commit-config.yaml` has:
```yaml
files: '^(src/bindings/python/flux|src/cmd|t/.*\\.py)'
exclude: "^(src/bindings/python/_flux/|src/bindings/python/flux/utils/)"
```
So it does appear that inclusion of python files under `t/` was intended. I think the double backslash is preventing this regex from matching though. I suppose we should fix this in a separate PR. I leave whether you keep the unrelated formatting change in this PR up to you.

Reply:
FYI - I have a PR ready that fixes `.pre-commit-config.yaml` and all the fallout...

Reply:
Hmmm, I guess I ran black on it, leading to that change.